comfy_panel/config.lua

```diff
@@ -82,6 +82,9 @@ end
 
 local function spaghett()
     local spaghetti = this.gui_config.spaghett
+    if spaghetti.noop then
```

Comment:
No idea what you are doing here - spaghett is not enabled unless a player admin enables it.

This can be removed.

Reply:
If this isn't here, then the spaghett module keeps re-enabling the logistic systems technology when you pass through
https://github.com/eric-anderson/ComfyFactorio/blob/75864b385c38d2cb191d22a3bed207858d7d1179/comfy_panel/config.lua#L83

Reply:
I'd still want to redo the trigger function instead of adding this.
But I understand now why you are adding it here.

Looks good for now.